libs/sdk-ui-charts/src/highcharts/chart/HighChartsRenderer.tsx

```diff
@@ -199,6 +199,17 @@ export default class HighChartsRenderer extends React.PureComponent<
         });
     }
 
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    private onChartSelection = (event: any): void => {
+        const chartWrapper = event.target.renderTo.parentElement;
+        const resetZoomButton = chartWrapper.closest(".gd-base-visualization").querySelector(".viz-zoom-out");
```

Comment:
This is not very React way of doing things. Can we use a state flag, and toggle it here, and use it in `renderZoomOutButton`? Or was this a performance decision?

Reply:
I try using a state flag, but when the state is set, the chart will be re-rendered also. So, instead of the chart will zoom in when we selected the zoom area but it doesn't, it still is normal.